src/compiler/diagnosticMessages.json

```diff
@@ -5322,6 +5322,14 @@
         "category": "Message",
         "code": 6398
     },
+    "Project '{0}' is out of date because buildinfo file '{1}' indicates that some of the changes are not emitted": {

```

Comment:
"were not emitted"?  I'm not sure what this is intended to convey.

Reply:
https://github.com/microsoft/TypeScript/pull/48784/files#diff-0558af289c76f66611354da63a14152fe5cd3a72b3c3e59eb880808efc2df442R1596 This is when buildInfo shows that there iis changeFileSet or affected files pending emit indicating  project emit was not complete.